airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -220,10 +220,48 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
 
         return available_streams
 
+    # set of standard stream names to check for custom object name collisions
+    STANDARD_STREAM_NAMES = {
+        "campaigns",
+        "companies",
+        "contact_lists",
+        "contacts",
+        "contacts_form_submissions",
+        "contacts_list_memberships",
+        "contacts_merged_audit",
+        "deal_pipelines",
+        "deals",
+        "deals_archived",
+        "email_events",
+        "email_subscriptions",
+        "engagements",
+        "engagements_calls",
+        "engagements_emails",
+        "engagements_meetings",
+        "engagements_notes",
+        "engagements_tasks",
+        "forms",
+        "form_submissions",
+        "goals",
+        "line_items",
+        "marketing_emails",
+        "owners",
+        "owners_archived",
+        "products",
+        "contacts_property_history",
+        "companies_property_history",
+        "deals_property_history",
+        "subscription_changes",
+        "tickets",
+        "ticket_pipelines",
+        "workflows",
+    }
+
     def get_custom_object_streams(self, api: API, common_params: Mapping[str, Any]):
         for entity, fully_qualified_name, schema, custom_properties in api.get_custom_objects_metadata():
             yield CustomObject(
                 entity=entity,
+                name=fully_qualified_name if entity in self.STANDARD_STREAM_NAMES else entity,
```

Comment:
could you add a comment with examples for fully_qualified_name and entity to show the difference and provide more clarity?

Reply:
Yes, done